src/EditorFeatures/Core.Wpf/Interactive/CSharpVBResetCommand.cs

```diff
@@ -112,33 +112,42 @@ internal static IEnumerable<int> GetNoConfigPositions(string arguments)
         /// <remarks>
         /// Accessibility is internal for testing.
         /// </remarks>
-        internal static bool TryParseArguments(string arguments, out bool initialize, out bool? is64bit)
+        internal static bool TryParseArguments(string arguments, out bool initialize, out InteractiveHostPlatform? platform)
         {
-            is64bit = null;
+            platform = null;
             initialize = true;
 
             var noConfigSpecified = false;
 
             foreach (var argument in arguments.Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries))
             {
-                switch (argument)
+                switch (argument.ToLowerInvariant())
                 {
-                    case "32":
-                        if (is64bit != null)
+                    case PlatformDesktop32:
+                        if (platform != null)
+                        {
+                            return false;
+                        }
+
+                        platform = InteractiveHostPlatform.Desktop32;
+                        break;
+
+                    case PlatformDesktop64:
+                        if (platform != null)
                         {
                             return false;
                         }
 
-                        is64bit = false;
+                        platform = InteractiveHostPlatform.Desktop64;
                         break;
 
-                    case "64":
-                        if (is64bit != null)
+                    case PlatformCore:
+                        if (platform != null)
                         {
                             return false;
                         }
 
-                        is64bit = true;
+                        platform = InteractiveHostPlatform.Core;
```

Comment:
Is there a 64/32 bit runtime option for Core?

Reply:
No, we currently use the .NET Core that's on PATH for VS process. Whatever bitness that is.

Reply:
Looks like .Net Core runtime is not a required component for Desktop workload? Some setup change in VS might be required

![image](https://user-images.githubusercontent.com/788783/86058806-4ea98200-ba16-11ea-9584-2a85f3d9f5ae.png)

Reply:
@genlu HAve you researched what setup changes need to be done to add this dependency?

Reply:
I don't really know if CoreClr is installed as part of VS core editor, this is more of a question. If we just want to make .NetCore 2.1 Runtime a required component, then I think all we need is a one line change here.
https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS?path=%2Fsrc%2FSetupPackages%2FGroup%2FWorkload%2FManagedDesktop%2Fworkload.swr&version=GBmaster&line=146&lineEnd=147&lineStartColumn=1&lineEndColumn=1&lineStyle=plain

@JoeRobich is the current acquisition champ, could you please help us figure out if this is the right way to do it? I think all we need is core runtime, we don't need the entire .Net Core SDK.

Reply:
This looks like the component contains runtime only, maybe we just need to add this as required in Desktop workload linked above.
https://devdiv.visualstudio.com/DevDiv/_git/VS?path=%2Fsrc%2FSetupPackages%2FGroup%2FComponent%2FDotNetCoreSDK%2Fcomponent.swr&version=GBmaster&line=13&lineEnd=14&lineStartColumn=1&lineEndColumn=1&lineStyle=plain

Reply:
Well, scratch all that, turns out .NET Core SDK (which contains 3.1 runtime) is part of the "ManagedDesktop.Prerequisites", I think we are good here. Just need to install the build from insertion validation to confirm this.
https://devdiv.visualstudio.com/DevDiv/_git/VS?path=%2Fsrc%2FSetupPackages%2FGroup%2FComponentGroup%2FManagedDesktop.Prerequisites%2Fcomponent.swr&version=GBmaster&line=30&lineEnd=31&lineStartColumn=1&lineEndColumn=1&lineStyle=plain

Reply:
Thanks!